docs/UserGuide.md

```diff
@@ -560,6 +595,16 @@ Searches for a public address.
 
 Format: `searchpa pa/PUBLIC_ADDRESS`
 
+
+<box type="tip" seamless>
+
+The field PUBLIC_ADDRESS is not cap sensitive.
+This command only searches the current list of public addresses displayed(eg if all contacts is displayed
```

Comment:
nit: (grammar) change "if all contacts is displayed" to "if all contacts are displayed"

Reply:
yes sir this shall be fixed